Start-AzerothCoreAutoBuilder.ps1

```diff
@@ -563,68 +568,63 @@ do {
+                Remove-Item (Join-Path $BuildFolder "\bin\Release") -Recurse -Force
             }
             #Build Server
+			
+	
             Set-Location 'C:\Program Files\CMake\bin'
             Write-Progress -Activity "Building Server" -Status "Compiling Source"
             Write-Information -MessageData "Compiling and building will take some time. Go have a beer!" -InformationAction Continue
-            $BuildArgs = "-G `"Visual Studio 16 2019`" -A x64 -DTOOLS=1 -S $BaseLocation -B $BuildFolder"
+            $BuildArgs = "-G ""Visual Studio 16 2019"" -A x64 -DTOOLS=1 -S $BaseLocation -B $BuildFolder"
             Start-Process -FilePath 'C:\Program Files\CMake\bin\cmake.exe' -ArgumentList $BuildArgs -Wait
             Write-Progress -Activity "Building Server" -Status "Final Build"
             $FinalArgs = "--build $BuildFolder --config Release"
             Start-Process -FilePath "C:\Program Files\CMake\bin\cmake.exe" -ArgumentList $FinalArgs -Wait
             Write-Progress -Activity "Building Server" -Status "Ready" -Completed
             # Check to ensure build finished
-            if ((Test-Path -Path "$BuildFolder\bin\Release\authserver.exe") -and (Test-Path -Path "$BuildFolder\bin\Release\worldserver.exe")) {
+            if ((Test-Path (Join-Path $BuildFolder "\bin\Release\authserver.exe")) -and (Test-Path (Join-Path $BuildFolder "\bin\Release\worldserver.exe"))) {
                 Write-Information -MessageData "Compile and build Successful! Continuing..." -InformationAction Continue
             } else {
                 Write-Information -MessageData "Compile and build failed.  Check cmake logs and try again." -InformationAction Continue
                 break
             }
 
             # Copy all conf.dist files to .conf
-            $DistFiles = Get-ChildItem -Path "$BuildFolder\bin\Release\configs" -Filter "*.dist"
+            $DistFiles = Get-ChildItem (Join-Path $BuildFolder "bin\Release\configs") -Filter "*.dist"
             foreach ($Dist in $DistFiles) {
                 $Conf = $Dist -replace ".{5}$"
-                Copy-Item -Path "$BuildFolder\bin\Release\configs\$Dist" -Destination "$BuildFolder\bin\Release\configs\$Conf"
+                Copy-Item (Join-Path $BuildFolder "\bin\Release\configs\$Dist") -Destination (Join-Path $BuildFolder "\bin\Release\configs\$Conf")
             }
 
-            # Copy all conf.dist files from mods to .conf if modules are found
-            if (Test-Path -path "$BuildFolder\bin\Release\configs\modules") {
-                $ModDistFiles = Get-ChildItem -Path "$BuildFolder\bin\Release\configs\modules" -Filter "*.dist"
-                foreach ($moddist in $ModDistFiles) {
-                    $Conf = $moddist -replace ".{5}$"
-                    Copy-Item -Path "$BuildFolder\bin\Release\configs\modules\$moddist" -Destination "$BuildFolder\bin\Release\configs\modules\$Conf"
-                }
-            }
```

Comment:
I like and understand your edits except this one.  Why delete this section? If any modules were downloaded the config files for each will need to be manually copied without this.